src/components/portal/portal.js

```diff
@@ -1,35 +1,63 @@
 import React from 'react';
-import ReactPortal from 'react-portal';
 import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
+import Browser from '../../utils/helpers/browser';
 
 class Portal extends React.Component {
-  static propTypes = {
-    /**
-     * Determine if the portal is visible or not
-     *
-     * @property open
-     * @type {Boolean}
-    */
-    open: PropTypes.bool,
 
+  static propTypes = {
     /**
      * The content of the portal.
      *
      * @property children
      * @type {Node}
      */
-    children: PropTypes.node
+    children: PropTypes.node,
+    /**
+     * The DomNode 
+     *
+     * @property node
+     * @type {Any}
+     */
+    node: PropTypes.any
+  }
+
+  componentWillUnmount() {
+    if (this.defaultNode) {
+      Browser.getDocument().body.removeChild(this.defaultNode);
+    }
+    this.defaultNode = null;
+  }
+
+  getDefaultDiv() {
+    
+    if (!this.props.node && !this.defaultNode && Browser.isDomAvailable()) {
+      this.defaultNode = Browser.getDocument().createElement('div');
+      this.defaultNode.classList.add('portal');
+      document.body.appendChild(this.defaultNode);
+    }
+    return this.defaultNode;
   }
 
   render() {
+    if (!Browser.isDomAvailable()) {
+      return null;
+    }
+
+    this.node = (this.props.node) ? this.props.node : this.getDefaultDiv();
     return (
-      <ReactPortal isOpened={ this.props.open }>
-        <div ref={ (node) => { this._portal = node; } }>
-          { this.props.children }
-        </div>
-      </ReactPortal>
+      <ReactPortal node={ this.node }>{this.props.children}</ReactPortal>
     );
   }
 }
 
+function ReactPortal(props) {
```

Comment:
any particular reason you've done this as a separate component, rather than just using `createPortal` method in the `render` function?